DigitalOcean Spaces (syd1).cyberduckprofile

```diff
@@ -0,0 +1,558 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2017 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>digitalocean-syd1</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>DigitalOcean Spaces (syd1)</string>
+        <key>Default Hostname</key>
+        <string>syd1.digitaloceanspaces.com</string>
+        <key>Hostname Configurable</key>
+        <false/>
+        <key>Port Configurable</key>
+        <false/>
+        <key>Username Placeholder</key>
+        <string>Access Key</string>
+        <key>Password Placeholder</key>
+        <string>Secret Key</string>
+        <key>Authorization</key>
+        <string>AWS4HMACSHA256</string>
+        <key>Region</key>
+        <string>syd1</string>
+        <key>Regions</key>
+        <array>
+            <string>syd1</string>
+        </array>
+        <key>Properties</key>
+        <array>
+            <string>s3.storage.class.options=STANDARD</string>
+        </array>
+        <key>Help</key>
+        <string>https://trac.cyberduck.io/wiki/help/en/howto/digitalocean</string>
```

Comment:
Please change to the redirect target `https://docs.cyberduck.io/protocols/s3/digitalocean/`

```suggestion
        <string>https://docs.cyberduck.io/protocols/s3/digitalocean/</string>
```